src/umpire/ResourceManager.cpp

```diff
@@ -20,6 +20,9 @@
 #include "umpire/strategy/DynamicPool.hpp"
 #include "umpire/strategy/FixedPool.hpp"
 #include "umpire/strategy/ZeroByteHandler.hpp"
+#if defined(UMPIRE_ENABLE_NUMA)
+#include "umpire/strategy/NumaPolicy.hpp"
+#endif
```

Comment:
It seems strange (to me) that the only change made to this file was to add a guarded include of NumaPolicy.hpp.  Is there someplace else further down in this file that should also be guarded?

Reply:
I think this got accidentally removed at some point. The `move` operation uses this class.

Reply:
Ah.  I guess we wern't building with NUMA very much.  Glad it has been added to CI!